test/unit/widgets/forms/__snapshots__/basic.test.ts.snap

```diff
@@ -2,67 +2,66 @@
 
 exports[`basicField adds the basic field to the container, if provided 1`] = `
```

Comment:
**thought:** So according to theses tests / snapshots only some styling changed and the layout switched to div-based. Looking at the changed code above this cannot be the whole truth.

**suggestion:** Cover all changes by tests, especially changes in behaviour.

Reply:
Well there were no tests for the autocomplete field... working on that.

Reply:
Tests done now